usama/sprint1/cdk.out/asset.3e5d1abf219cc4b44934567e5e5e989698413c159b2783d4d5842824fd99066e/constants.py

```diff
@@ -0,0 +1 @@
+URLS_TO_MONITOR = "https://www.skipq.org/"
```

Comment:
- for the comment mentioned in web_health_publisher, you can update this constant to be an arrary

Reply:
Resolved.